jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/EditFunctionDialog.kt

```diff
@@ -308,12 +313,25 @@ class EditFunctionDialog(
 
         override fun doAction(e: ActionEvent?) {
             super.doAction(e)
+            saveSettings()
             if (validation() == null) {
                 performUpdate()
             }
         }
     }
 
+    private fun loadSettings() {
+        val updateSettings = UpdateLambdaSettings.getInstance(project)
+        view.sourceBucket.selectedItem = updateSettings.bucketName(name)
+        view.buildInContainer.isSelected = updateSettings.useContainer(name) ?: false
+    }
+
+    private fun saveSettings() {
+        val updateSettings = UpdateLambdaSettings.getInstance(project)
```

Comment:
we probably want it bound by lambda rather than by project no?

Reply:
how will this work if the customer has switched accounts?

I think there's a bit of a hierarchy here - for a given account, pick the bucket based on :
1) lambda if the setting exists
2) fall back to project-wide setting if it exists
3) fall back to app-wide setting if it exists

Reply:
We could make it a `applicationService`, but it is currently bound by Lambda name and project, which matches how we do other settings